CHANGELOG.md

```diff
@@ -17,6 +17,13 @@
 - [BUGFIX] Minor corrections and spelling issues have been fixed in the Overview
   documentation. (@amckinley)
 
+- [BUGFIX] The `agent_prometheus_active_configs` metric will now properly track
```

Comment:
This only broke in instance sharing mode, right? Let's add it on the changelog as it is really don't as bad as it sounds.